### ISSUE TITLE
confidence threshold

### ISSUE THREAD

**Issue by ChethanN01** [tomas-gajarsky/facetorch#45]
can i get the confidence measure of the prediction,
the neutral is being detected with other emotion ,  any idea on how i can fine tune the model.

**Comment by tomas-gajarsky** [comment]
The confidence of each facial expression can be accessed from the response for the first face as follows:
`response.faces[0].preds["fer"].logits`
The list of all facial expressions that the default FER model works with can be found in the [config](https://github.com/tomas-gajarsky/facetorch/blob/main/conf/analyzer/predictor/fer/efficientnet_b2_8.yaml#L36).


The FER model selects a single facial expression that has the confidence, therefore it is not possible that two or more emotions are detected for a single face. The predicted facial expression of the first face can be accessed from the response as follows: 
`response.faces[0].preds["fer"].label
`

In order to fine-tune the model on your own data, you would have to adapt the original codebase [linked in the README](https://github.com/tomas-gajarsky/facetorch#facial-expression-recognition-fer).
